cli/src/helpers/git.ts

```diff
@@ -96,6 +99,7 @@ export const initializeGit = async (projectDir: string) => {
 
   // We're good to go, initializing the git repo
   try {
+    spinner.stop();
```

Comment:
should this be here? @c-ehrlich

Reply:
probably not, i think its leftover from trying stuff

Reply:
removed this in [5c95429](https://github.com/t3-oss/create-t3-app/pull/621/commits/5c95429f148b5f1b5f8ef7c59b9b8078997a49b9)